package.json

```diff
@@ -72,7 +73,8 @@
     "chalk": "^2.1.0",
     "convert-source-map": "^1.6.0",
     "extract-from-css": "^0.4.4",
-    "source-map": "^0.5.6",
```

Comment:
@JessicaSachs  this package is used here https://github.com/vuejs/vue-jest/blob/0b0d5e2dcd28da728bc0928efdfa6817df6113e7/lib/generate-source-map.js#L2